### ISSUE TITLE
Problem in semantic highlighting C standard library functions

### ISSUE THREAD

**Issue by iosmanthus** [cquery-project/emacs-cquery#34]
In `vscode`, C standard library functions are highlighted as ordinary functions, but in Emacs with emacs-cquery they are not highlighted but others are highlighted properly
![screenshot from 2018-04-06 18-31-03](https://user-images.githubusercontent.com/16307070/38417485-153acdf8-39cb-11e8-97b6-f2dfe4091c64.png)
.

**Comment by innerout** [comment]
I dont think that is a cquery feature.Probably vscode has a plugin that highlights the standard library functions.
